### PR TITLE
v0.21.0

### PR DESCRIPTION
**Pull request by eladb**
See CHANGELOG

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,41 @@
...
+* **codebuild:** Rename includeBuildID property of S3BucketBuildArtifacts ([#1354](https://github.com/awslabs/aws-cdk/issues/1354)) ([84eb7ad](https://github.com/awslabs/aws-cdk/commit/84eb7ad)), closes [#1347](https://github.com/awslabs/aws-cdk/issues/1347)
+* **toolkit:** scrutiny dialog should fail with no tty ([#1382](https://github.com/awslabs/aws-cdk/issues/1382)) ([478a714](https://github.com/awslabs/aws-cdk/commit/478a714)), closes [#1380](https://github.com/awslabs/aws-cdk/issues/1380)
+
+
+### Features
+
+* **aws-codebuild:** change the API of GitHub and BitBucket Sources. ([#1345](https://github.com/awslabs/aws-cdk/issues/1345)) ([9cebf0d](https://github.com/awslabs/aws-cdk/commit/9cebf0d))
+* add "engines.node" key to all packages ([#1358](https://github.com/awslabs/aws-cdk/issues/1358)) ([b595cf0](https://github.com/awslabs/aws-cdk/commit/b595cf0)), closes [#1337](https://github.com/awslabs/aws-cdk/issues/1337)
```
Comment (RomainMuller) [inline CHANGELOG.md]: That is more of a `chore` I think.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,41 @@
...
+* deprecate "cloudformation" namespace in favor of "CfnXxx" ([#1311](https://github.com/awslabs/aws-cdk/issues/1311)) ([d20938c](https://github.com/awslabs/aws-cdk/commit/d20938c)), closes [#878](https://github.com/awslabs/aws-cdk/issues/878) [awslabs/jsii#283](https://github.com/awslabs/jsii/issues/283) [awslabs/jsii#270](https://github.com/awslabs/jsii/issues/270)
```
Comment (RomainMuller) [inline CHANGELOG.md]: Maybe we should mention when we'll drop the old names...

Comment (eladb) [reply CHANGELOG.md]: The deprecation message says "future release". We decided not to commit to that yet.

File: CHANGELOG.md (hunk 1 above)
Comment (eladb) [reply CHANGELOG.md]: You are right. I'll remove from the changelog

Comment (eladb) [reply CHANGELOG.md]: You know what, I'll leave this. Since it does affect users, I think it should be indicated there

File: CHANGELOG.md (hunk 2 above)
Comment (RomainMuller) [reply CHANGELOG.md]: Oh right. Sweet.

**Pull request merged**